lib/browser_launcher.js

```diff
@@ -239,6 +240,19 @@ function browsersForPlatform() {
         supported: findableByWhichOrModule
       }
     ];
+
+    // If dbus-launch is present, wrap google-chrome launch
+    // to prevent a race condition in Google Chrome
+    // Discussion here: https://github.com/angular/protractor/issues/2419
+    if (whereis('dbus-launch')) {
```

Comment:
We already have a function, which does this. Could you replace `node-whereis` with https://github.com/testem/testem/blob/master/lib/fileutils.js#L19?